docs/architecture/solana/migration-faq.mdx

```diff
@@ -27,19 +25,19 @@ switching to `devnet` and back to `mainnet-beta`, or updating the app.
 
 ## Where Can I See My Hotspot's Activity And Rewards?
 
-Oracle Data is available to application developers such as [Helium Geek](https://heliumgeek.com/)
-and [Hotspotty](https://app.hotspotty.net/), which are building interfaces to monitor Hotspot
+Oracle Data is available to application developers such as [Helium Geek](https://heliumgeek.com/), 
+[Moken](https://explorer.moken.io/), Relay[https://app.relaywireless.com/explorer] and [Hotspotty](https://app.hotspotty.net/), who have built interfaces to monitor Hotspot
 activity and rewards.
 
 ## What Is The IOT Token?
 
-Instead of earning HNT, Hotspots will earn IOT rewards for their PoC and data transfer. IOT can be
-swapped for HNT in the Helium Wallet app or on an exchange. Find more info about IOT
+Instead of earning HNT, LoRaWAN Hotspots will earn IOT rewards for their PoC and data transfer. 
+IOT can be swapped for HNT in the Helium Wallet app or on an exchange. Find more info about IOT
 [here](/tokens/iot-token).
```

Comment:
We should avoid links with non-descriptive labels. There are a few throughout this document.
See: https://github.com/helium/docs/issues/1690